.pylintrc

```diff
@@ -335,10 +335,6 @@ single-line-class-stmt=no
 # else.
 single-line-if-stmt=no
 
-string-quote=double-avoid-escape
```

Comment:
We would probably want to preserve this config for flake8 quotes, so I've checked the docs and found something like this:
<img width="796" alt="Screenshot 2024-07-16 at 08 31 50" src="https://github.com/user-attachments/assets/ef476302-f240-4e6d-b672-353291c2ff3e">

Reply:
So our conventions are documented in this config and they are intentional.

Reply:
Updated 👍